autoload/phpactor.vim

```diff
@@ -435,14 +435,38 @@ endfunction
 " RPC -->-->-->-->-->--
 """""""""""""""""""""""
 
+function! s:searchDirectoryUpwardForRootPatterns(initialDirectory, rootPatterns, fallbackDirectory)
+  let l:directory = a:initialDirectory
+  while index(g:phpactorGlobalRootPatterns, l:directory) < 0
+    for l:pattern in a:rootPatterns
+      if (filereadable(l:directory .'/'. l:pattern))
+        return l:directory
+      endif
+    endfor
+    let l:directory = fnamemodify(l:directory, ':h')
```

Comment:
this would still detect the `/home` is it had a global pattern in it or?

Reply:
See comment below